backend/src/schemas/slide-object.schema.ts

```diff
@@ -0,0 +1,227 @@
+import mongoose from 'mongoose';
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { User } from './user.schema';
+import { Slide } from './slide.schema';
+
+export type SlideObjectDocument = SlideObject & mongoose.Document;
+
+export class Shadow {
+  color: string;
+  blur: number;
+  offsetX: number;
+  offsetY: number;
+}
+
+@Schema({ timestamps: true })
+export class SlideObject {
+
+  // relations
```

Comment:
I would suggest moving the relations to the very end, as it is for the other schemas